subsys/usb/class/bluetooth.c

```diff
@@ -32,11 +33,19 @@ NET_BUF_POOL_DEFINE(tx_pool, CONFIG_BT_HCI_CMD_COUNT, CMD_BUF_SIZE,
 		    sizeof(u8_t), NULL);
 
 /* ACL data TX buffers */
+#if defined(CONFIG_USB_DEVICE_BLUETOOTH_VS_H4)
+#if defined(CONFIG_BT_CTLR_TX_BUFFERS)
+#define ACL_BUF_COUNT MAX(CONFIG_BT_HCI_CMD_COUNT, CONFIG_BT_CTLR_TX_BUFFERS)
```

Comment:
I don't follow this. Why is it MAX in this expression.
If they are allocated from the same pool, shouldn't they be added?

Reply:
Yep, it be the sum of commands and acl buffers.